tornettools/tornettools

```diff
@@ -288,13 +288,26 @@ def main():
         default=0.1)
 
     generate_parser.add_argument('-t', '--torperf_scale',
-        help="""A fractional number representing the ratio of the number of Perf clients to the
+        help="""DEPRECATED. Use torperf_n instead.
+            A fractional number representing the ratio of the number of Perf clients to the
```

Comment:
I'm OK with removing - it's cleaner to have only one way to tweak the perf clients and I think the num approach is a bit easier to understand than the scale.

Happy to bump the major version number if we want to stay consistent with semantic versioning, but maybe that should wait until we're done with the various tweaks you've been making for the congestion control work.